tests/Skills/Parent/ParentBot.cs

```diff
@@ -29,11 +26,14 @@ public class ParentBot : ActivityHandler
         private readonly string _toBotId;
         private readonly string _fromBotId;
         private readonly string _connectionName;
+        private readonly SkillsHelper _skillsHelper;
+
+        private readonly HashSet<string> userinput = new HashSet<string>(StringComparer.InvariantCultureIgnoreCase) { "login", "skill login", "skill logout" };
```

Comment:
I understand what you're doing here, but when I looked at the [code below](https://github.com/microsoft/botbuilder-dotnet/pull/3546/files#diff-8286b79397c96c4acfaf34d6963baeddR62) I did a double take.

If the team decides to go with the HashSet, shouldn't `userinput` be `_userInput`?